transport/http/outbound.go

```diff
@@ -52,6 +52,14 @@ type outbound struct {
 	URL    string
 }
 
+func (o outbound) Start() error {
+	return nil // nothing to do
```

Comment:
Same as I said before - I think the user benefit outweighs the sync cost - make things dummy-proof :)